Makefile

```diff
@@ -10,6 +10,22 @@ test:
 	@echo "+++ Running tests"
 	bundle exec rake spec
 
+#: run the lint checks
+lint:
```

Comment:
Should we all the missing `.PHONY` entries for the makefile targets that don't target files?